Sources/Web3Core/KeystoreManager/BIP32Keystore.swift

```diff
@@ -216,6 +216,54 @@ public class BIP32Keystore: AbstractKeystore {
         try encryptDataToStorage(password, data: serializedRootNode, aesMode: self.keystoreParams!.crypto.cipher)
     }
 
+    /// Fast generation addresses for current account
```

Comment:
Why "Fast"? 
Is there a slow version? Just would like to understand it's fast compared to which version of this algorithm.

Reply:
Faster than generate all this wallets with createNewCustomChildAccount

Reply:
Ah, okay.
I suggest you update the documentation comment to reflect that information.

Reply:
Please, also take into consideration my previous comments from yesterday regarding the documentation for this function.